### PR TITLE
cluster: remove store limit collect logic

### LINKED ISSUE #6833: store-limit:  repeated modify the store limit value
## Bug Report
https://github.com/tikv/pd/pull/6444 this pr brings the problem

### What did you do?
nothing

### What did you expect to see?
store limit don't change 
### What did you see instead?
[2023/07/24 06:06:50.455 +00:00] [INFO] [store_limiter.go:69] ["change store region remove limit for cluster"] [state=normal] [rate=32]
[2023/07/24 06:06:50.455 +00:00] [INFO] [store_limiter.go:69] ["change store region remove limit for cluster"] [state=normal] [rate=32]
### What version of PD are you using (`pd-server -V`)?
master

### PR DESCRIPTION
**Pull request by bufferflies**
### What problem does this PR solve?

Issue Number: Close #6833

### What is changed and how does it work?

```commit-message
We have remove the auto store limit stratery, so the store limit collect is unused.
In past, the collect will run auto not depence the configuration.
```

### Check List

Tests

- Unit test
- Integration test

Code changes

Side effects

Related changes

### Release note

```release-note
None.
```

### CODE REVIEW

File: server/cluster/store_limiter.go [hunk 1]
```diff
@@ -47,42 +44,6 @@ func NewStoreLimiter(opt *config.PersistOptions) *StoreLimiter {
...
-	for i := LoadStateNone; i <= LoadStateHigh; i++ {
-		if i == state {
-			clusterStateCurrent.WithLabelValues(state.String()).Set(1)
-			continue
-		}
-		clusterStateCurrent.WithLabelValues(i.String()).Set(0)
-	}
-}
-
 func (s *StoreLimiter) calculateRate(limitType storelimit.Type, state LoadState) float64 {
```
Comment (rleungx) [inline server/cluster/store_limiter.go:47]: remove it?

Comment (bufferflies) [reply server/cluster/store_limiter.go:47]: yes, there are no code to depence it.

Comment (rleungx) [reply server/cluster/store_limiter.go]: BTW, /stores/limit/scene should be removed since we have no any other scene

**Review by nolouch — APPROVED** [review]

**Pull request merged**